roles/custom/matrix-bridge-mautrix-bluesky/defaults/main.yml

```diff
@@ -31,6 +31,8 @@ matrix_mautrix_bluesky_homeserver_address: ""
 matrix_mautrix_bluesky_homeserver_domain: '{{ matrix_domain }}'
 matrix_mautrix_bluesky_appservice_address: 'http://matrix-mautrix-bluesky:29340'
 
+matrix_mautrix_bluesky_msc4190_enabled: "{{ matrix_bridges_msc4190_enabled }}"
```

Comment:
Not sure if I should really do the global flag or just do :
matrix_mautrix_bluesky_msc4190_enabled: "{{ matrix_authentication_service_enabled &&  matrix_mautrix_bluesky_encryption_enabled}}"